### ISSUE TITLE
Providers rely on annotation inheritance in JAX-RS

### ISSUE THREAD

**Issue by berezovskyi** [eclipse/lyo.core#67]
https://stackoverflow.com/a/25924310/464590 says that this behaviour is a MAY for a JAX-RS implementation and this bit seems to differ for Wink and Jersey.

**Comment by berezovskyi** [comment]
cc @jadelkhoury @jamsden

**Comment by berezovskyi** [comment]
proposed solution: remove all JAX-RS annotations from abstract classes.

**Comment by jamsden** [comment]
Agreed, and revert the change I did to AbstractOslcRdfXmlProvider.getSerializtionLanguage() which was a temporary patch to work around the issue.

We should make sure however that the resulting annotations reproduce the behavior that was supported through Wink's approach to annotation inheritance to avoid introducing bugs.
